utils/currencies.ts

```diff
@@ -21,7 +21,7 @@ export const formatValueWithFixedDecimals = (
   const smallNumberString = `<${smallestNumber.toFixed(fixedDecimals)}`;
   const isVerySmallNumber = Number(value) < smallestNumber;
 
-  if (isAnInteger) return value;
+  if (isAnInteger) return value.toString();
```

Comment:
why?

Reply:
We were returning `number | string` type, let's just return string since in the worst case this function returns `<0.01` which is a sting. So this function only returns `string`.

Reply:
thanks